.gitignore

```diff
@@ -1,5 +1,80 @@
-/.classpath
-/.project
-/.settings
-/bin
-*.class
\ No newline at end of file
+### Java template
```

Comment:
Would it be better to limit this file to only exclude the types of things that would be encountered while developing this plugin? It looks like there is a lot of additional things that may never get used.